### PR TITLE
979 Font style for Action text

### LINKED ISSUE #979: 'Action' component should be Noto Sans 400/regular, not semibold
**Describe the bug**
The font-family of the "Pre-fill my details" link on page 1 of the submission wizard is `Noto Sans Semi-bold` but should be regular (400 weight) `Noto Sans`

**To Reproduce**
Go to https://staging--xpub.elifesciences.org/ and create a new submission. Check the text above the First Name input of the form. 

**Expected behavior**
Regular font for the blue link

**Screenshots**
image

**Desktop (please complete the following information):**
Chrome  70.0.3538 on Mac OS Mojave

**Smartphone (please complete the following information):**
Also seen in Chrome on iPhone 6

**Additional context**
This relates to #915 where @jsms90 worked on changes to the regular `button` style, which affected the `link` style. It appears to have been corrected in other instances (e.g. on the login page) but the `link` in the wizard is still semibold

### PR DESCRIPTION
**Pull request by diversemix**
#### Background

Fix font style for text with actions

#### Any relevant tickets

Closes #979

#### How has this been tested?
**Reviewers** ensure that these test requirements are also reviewed.

No tests written

#### UI Changes

@chugginselifesciences  has seen on my machine

### CODE REVIEW

**Comment by hem-brahmbhatt** [comment]
We can export an `Action` fragment of CSS [here] and tweak the existing Pubsweet `Action` instead. The `ActionText` component is basically the same as the `Action` component, just with a different font, so it doesn't seem justified to create a brand new component for it.

**Comment by diversemix** [comment]
@damnhipster Action is used in other places too (e.g. `PersonPod.js` and re-styled locally as `SmallAction`) ... would we 'undo' the styling in here?

**Comment by hem-brahmbhatt** [comment]
`SmallAction` doesn't get exported as a component, it's used internally for the `PersonPod`.

Although, thinking about this a second time, I think `ActionText` makes more sense than Pubsweet's `Action` component, which is a confusing name for a component... So I'm happy to go with this.

**Review by hem-brahmbhatt — APPROVED** [review]

File: app/components/ui/atoms/ActionText.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+A portion of text that has an action associated with clicking it.
+
+```js
+      <ActionText onClick={alert('do action')}>
```
Comment (hem-brahmbhatt) [inline app/components/ui/atoms/ActionText.md]: `onClick` should be a function.

The alert pops up as soon as you open the styleguide currently.

Comment (diversemix) [reply app/components/ui/atoms/ActionText.md]: Ooops

**Comment by diversemix** [comment]
Thanks Hem ... Chris also reviewed in the review environment :+1:

**Pull request merged**